Automattic/wp-calypso
# Editor: Remove the convert to blocks dialog from iframed editor

#### Changes proposed in this Pull Request

* Removes the 'Convert to blocks' dialog that appears whenever a user opens Classic content in iframed editor. This prompt is being [replaced with a 'Convert to blocks' block toolbar option](https://github.com/Automattic/wp-calypso/issues/43626).

#### Testing instructions

*  Check out PR to local calypso dev env
* Create some content in classic editor
* Switch to block editor for that site
* Open classic content in block editor and check convert dialog does not show and that no errors thrown

Before:

![conver-blocks-before](https://user-images.githubusercontent.com/3629020/86544253-59fe2f80-bf79-11ea-9f0c-19e60d812668.gif)

After:

![convert-prompt-after](https://user-images.githubusercontent.com/3629020/86544259-671b1e80-bf79-11ea-96bc-acbe80d5134b.gif)


Part of #43626
